e2e/motoko/Main.mo

```diff
@@ -98,7 +98,7 @@ shared ({ caller = installer }) actor class Main() {
                 switch result {
                     case (#Consistent(#Ok _)) {};
                     case (#Consistent(#Err err)) {
-                        Debug.trap("received error for " # method # ": " # debug_show err);
+                        Debug.trap("received error for " # name # " " # method # ": " # debug_show err);
```

Comment:
How about renaming `name` to something more explicit?
In my opinion, `debug name` is also not very clear.

Reply:
Sure; that's a good point. Updated.